.vscode/tasks.json

```diff
@@ -0,0 +1,75 @@
+{
+  "version": "2.0.0",
+  "tasks": [
+	{
+		"type": "npm",
+		"script": "start",
+		"path": "newIDE/app/",
+		"group": {
+			"kind": "build",
+			"isDefault": true
+		},
+		"problemMatcher": [],
+		"label": "Start development server",
+		"detail": "Starts the GDevelop development server"
+	},
+	{
+		"type": "npm",
+		"script": "build",
+		"path": "GDevelop.js/",
+		"group": "build",
+		"problemMatcher": [],
+		"label": "Build GDevelop.js",
+		"detail": "Builds GDCore for newIDE"
+	},
+	{
+		"type": "npm",
+		"script": "format",
+		"path": "newIDE/app/",
+		"problemMatcher": [],
+		"label": "Format newIDE",
+		"detail": "Formats via prettier the javascript in the newIDE/app directory."
+	},
+	{
+		"type": "npm",
+		"script": "test",
+		"path": "GDJS/",
+		"group": {
+			"kind": "test",
+			"isDefault": true
+		},
+		"problemMatcher": [],
+		"label": "Run newIDE tests",
+		"detail": "Run tests for newIDE"
```

Comment:
Be consistent for periods at the end of the sentence (some have it, some don't) of the `detail` field.